Dockerfile

```diff
@@ -197,7 +200,7 @@ RUN mkdir /var/run/fetchmail && chown fetchmail /var/run/fetchmail
 # Configures Postfix
 COPY target/postfix/main.cf target/postfix/master.cf /etc/postfix/
 COPY target/postfix/header_checks.pcre target/postfix/sender_header_filter.pcre target/postfix/sender_login_maps.pcre /etc/postfix/maps/
-RUN echo "" > /etc/aliases
+RUN : >/etc/aliases
```

Comment:
Why not just `RUN >/etc/aliasses` which is even shorter ;-)

Reply:
True indeed, and that would work, but I think we talked about this a while ago :D This nooo `:` makes the intention clear and is idiomatic Bash :)

Reply:
Now that you mention it, I can remember 😇  But _personally_ I don't like using functionless commands like in that case and don't see how that makes the intention any clearer. Using something like `while :; do foo; done` is a better noop use case.
But in the end, it's not worth discussing about +/- 2 bytes 😅 👍

Reply:
+1